src/Kanbas/Courses/Quizzes/Editor/Details/index.tsx

```diff
@@ -1,11 +1,245 @@
     return (
-        <>
-            <Form>
-            </Form>
-        </>
+        <Form>
+            <Form.Group className='mb-3 w-50' controlId="formQuizName">
+                <Form.Control
+                    type="text"
+                    className="form-control"
+                    value={quizDetails.title}
+                    placeholder="Quiz Name"
+                    onChange={(e) => dispatch(setQuiz({
+                        ...quizDetails, title: e.target.value
+                    }))}
+                    onFocus={(e) => e.target.style.textAlign = 'left'}
+                />
+            </Form.Group>
+
+            <Form.Group className='mb-3' controlId='formQuizDescription'>
+                <Editor
+                    apiKey={API_KEY}
+                    onEditorChange={(newValue, editor) => onEditorInputChange(newValue, editor)}
+                    onInit={(evt, editor) => setText(editor.getContent({ format: "text" }))}
+                    value={value}
+                    initialValue={quizDetails.description === null || quizDetails.description === ""
+                        ? "Write quiz instructions here..."
+                        : quizDetails.description}
+                    init={{
+                        height: 500,
+                        plugins: [
+                            'advlist', 'autolink', 'lists', 'link', 'image', 'charmap', 'preview',
+                            'anchor', 'searchreplace', 'visualblocks', 'code', 'fullscreen',
+                            'insertdatetime', 'media', 'table', 'code', 'help', 'wordcount'
+                        ],
+                        toolbar: 'undo redo | blocks | ' +
+                            'bold italic forecolor | alignleft aligncenter ' +
+                            'alignright alignjustify | bullist numlist outdent indent | ' +
+                            'removeformat | help',
+                        content_style: 'body { font-family:Helvetica,Arial,sans-serif; font-size:14px }'
+                    }}
+                />
+            </Form.Group>
+            <div>
+                <Form.Group className="row mb-3" controlId="formQuizType">
+                    <div className="col-3">
+                        <Form.Label className="float-end mt-1">
+                            Quiz Type
+                        </Form.Label>
+                    </div>
+                    <div className="col-6 mr-auto">
+                        <Form.Select
+                            aria-label="Quiz Type"
+                            value={quizDetails.quizType}
+                            onChange={(e) => dispatch(setQuiz({
+                                ...quizDetails, quizType: e.target.value
+                            }))}
+                        >
+                            {quizTypes.map((key, index) => (
+                                <option key={index} value={eQuizType[key as keyof typeof eQuizType]}>
```

Comment:
The enum name should not be displayed in its raw format. It should be converted to Title Case with spaces instead of underscores.